src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+@import "../variables/variables";
+
+.calendar {
+  margin: 100px auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 710px;
+  align-items: center;
+
+  &__day {
+    @include field-size($fields-size);
+    border: 1px solid black;
+    box-sizing: border-box;
+    background-color: #eee;
+    font-family: $font-family;
+    font-size: $font-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: all ease-in-out 0.5s;
```

Comment:
Don't set transition to all. List all styles that you apply transition to. It's more readable and you know exactly what is affected by it.